torch/nested/_internal/ops.py

```diff
@@ -1678,6 +1678,31 @@ def embedding_default(func, *args, **kwargs):
     )
 
 
+@register_jagged_func(
+    torch.ops.aten.embedding_dense_backward.default,
+    "self: jt, grad_output: jt, num_weights: any, padding_idx: any, scale_grad_by_freq: any",
+)
+def embedding_dense_backward(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(  # type: ignore[misc]
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    indices: NestedTensor = new_kwargs.pop("indices")
+    num_weights: int = new_kwargs.pop("num_weights")
+    grad_output: NestedTensor = new_kwargs.pop("grad_output")
+    if new_kwargs["scale_grad_by_freq"]:
+        raise NotImplementedError
+
+    check_ragged_dim_same(func, indices, "self", grad_output, "grad_output")
+    out = torch.zeros(num_weights, grad_output.size(-1))
+    src = grad_output._values
+    indices = indices._values
+    indices = indices[indices != new_kwargs["padding_idx"]]
+    indices = indices.long().unsqueeze(1).expand(-1, src.size(1))
+    out.scatter_add_(dim=0, index=indices, src=src)
```

Comment:
Rather than reimplement the backward behavior manually, I think it's possible to simply redispatch to the dense implementation (i.e. call `func` with unwrapped buffers). I think this should do the trick:
```python
@register_jagged_func(
    torch.ops.aten.embedding_dense_backward.default,
    "grad_output: jt, indices: jt, num_weights: any, padding_idx: any, scale_grad_by_freq: any",
)
def embedding_dense_backward_default(func, *args, **kwargs):
    _, new_kwargs = normalize_function(  # type: ignore[misc]
        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
    )

    indices = new_kwargs.pop("indices")
    grad_output = new_kwargs.pop("grad_output")
    return func(grad_output._values, indices._values, **new_kwargs)
```